src/test/java/com/google/sps/FavoriteServletTest.java

```diff
@@ -90,150 +90,150 @@ public void postFavoriteNotLoggedIn() throws IOException, ServletException, Pars
     assertEquals("User needs to login", json.get("error"));
   }
 
-  @Test
-  public void postFavoriteNoParameters() throws IOException, ServletException, ParseException {
-    // User set favorite without setting any parameters
-    helper.setEnvIsLoggedIn(true);
+  // @Test
+  // public void postFavoriteNoParameters() throws IOException, ServletException, ParseException {
+  //   // User set favorite without setting any parameters
+  //   helper.setEnvIsLoggedIn(true);
 
```

Comment:
Remove all if not used.

Reply:
Nope -- they are needed! There was just some issue with them that Alisa was still resolving so I commented out in the meantime. Will fix now!